src/components/Checkbox/style.css

```diff
@@ -0,0 +1,3 @@
+.container :global(.ant-checkbox-wrapper) {
+width: 12px !important;
```

Comment:
is this !important needed?

Reply:
pretty sure, I'll double check but some of these ant components will override styling unless you use the !important tag